ui/changes/src/index.js

```diff
@@ -237,6 +241,31 @@ function addRow(bugSummary) {
   risk_column.append(risk_text);
 }
 
+async function populateVersions() {
+  var versionSelector = document.getElementById("releaseVersions")
+
+  let data = await landingsData;
+
+  var allVersions = new Set();
+    for (let bug of Object.values(data)) {
+      bug.some(item => {
+        if (item.versions.length) {
+          allVersions.add(...item.versions)
+        }
+      })
```

Comment:
`some` is a function to check if at least one element of an array matches a given condition.
In this case, `forEach`, or a plain for loop, would be better.

Note: `some` and `forEach` are equivalent functionally here because your function is never returning a truthy value, but using `some` is a bit of a stretch.